logbook-logstash-logback-encoder/src/main/java/org/zalando/logbook/lle/AutodetectPrettyPrintingMarker.java

```diff
@@ -0,0 +1,46 @@
+package org.zalando.logbook.lle;
```

Comment:
Any idea for a more speaking package name?

Reply:
org.zalando.logbook.logback.logstash or org.zalando.logbook.logstash??

Reply:
> org.zalando.logbook.logstash

I'd go with that. Module name (directory/artifactId) would then be `logbook-logstash`?

Reply:
Yeah, lets go with that.